api/envoy/config/route/v3/route_components.proto

```diff
@@ -1329,6 +1329,10 @@ message RetryPolicy {
   // details.
   repeated RetryHostPredicate retry_host_predicate = 5;
 
+  // Retry options predicates that will be applied prior to retrying a request. These predicates
+  // allow customizing request behavior between retries.
+  repeated core.v3.TypedExtensionConfig retry_options_predicates = 12;
```

Comment:
Is there a `[#extension-category:]` that this field can be associated with?

Reply:
I think if I add this the docs will break because there are no built in extensions. Let me add a comment/TODO here to add a category if/when there are built-in extensions?

Reply:
Yeah confirmed this doesn't work. I will add a TODO.